src/pages/upgrades/get-involved/index.js

```diff
@@ -355,41 +338,19 @@ const GetInvolvedPage = ({ data, location }) => {
           <Translation id="page-upgrades-get-involved-run-clients-production" />
         </h3>
         <StyledCardGrid>
-          {clients
-            .filter((client) => client.isProductionReady)
-            .map((client, idx) => (
-              <ProductCard
-                key={idx}
-                url={client.url}
-                background={client.background}
-                image={getImage(client.image(isDarkTheme))}
-                name={client.name}
-                description={client.description}
-                alt={translateMessageId(client.alt, intl)}
-                githubUrl={client.githubUrl}
-                hideStars={true}
-              />
-            ))}
-        </StyledCardGrid>
-        <h3>
-          <Translation id="page-upgrades-get-involved-run-clients-experimental" />
-        </h3>
-        <StyledCardGrid>
-          {clients
-            .filter((client) => !client.isProductionReady)
-            .map((client, idx) => (
-              <ProductCard
-                key={idx}
-                url={client.url}
-                background={client.background}
-                image={getImage(client.image(isDarkTheme))}
-                name={client.name}
-                description={client.description}
-                alt={translateMessageId(client.alt, intl)}
-                githubUrl={client.githubUrl}
-                hideStars={true}
-              />
-            ))}
+          {clients.map((client, idx) => (
```

Comment:
This includes Lodestar as a production-ready consensus client. Is that correct?

Reply:
I believe its production ready after doing some looking around from Tim's comment yeah.

Reply:
Curious to hear from @carlbeek (& maybe others) on this. The PR on the Staking Launchpad hasn't yet been merged: https://github.com/ethereum/staking-launchpad/pull/436

Reply:
Yeah also curious on the status of this. Otherwise the rest of this looks great!

Reply:
Some Discord conversation leans towards identifying Lodestar as under review and audit. I have made this adjustment and sorted Lodestar to the end in the meantime, looking forward to removing this =)